src/asmcnc/apps/maintenance_app/screen_maintenance.py

```diff
@@ -576,6 +578,17 @@ def on_pre_enter(self):
         self.spindle_settings_widget.spindle_cooldown_speed.text = str(self.m.spindle_cooldown_rpm)
         self.spindle_settings_widget.rpm_override = self.m.spindle_cooldown_rpm_override
 
+        # Only show SC2 options if machine supports it
+        self.spindle_settings_widget.spindle_brand.values = self.spindle_settings_widget.brand_list_sc1
+        try:
+            # Check if $51 exists
+            self.m.s.setting_51
+            # Check if PRO+ is activated
+            if os.path.exists('/home/pi/proplus.txt'):
+                self.spindle_settings_widget.spindle_brand.values = self.spindle_settings_widget.brand_list_sc2
```

Comment:
using append() might have worked better here than having two lists (provided the kivy dropdown values allows it), but it doesn't need to block merge :)